concept/type/relationtype.feature

```diff
@@ -213,6 +213,29 @@ Feature: Concept Relation Type and Role Type
     Then relation(parentship) get role(parent) is abstract: true
     Then relation(parentship) get role(child) is abstract: true
     Then relation(parentship) create new instance; throws exception
+    When session opens transaction of type: write
+    When relation(parentship) set abstract: false
+    When put relation type: fathership
+    When relation(fathership) set supertype: parentship
+    When relation(fathership) set relates role: father as parent
+    When relation(fathership) set relates role: father-child as child
+    When put entity type: person
+    When entity(person) set plays role: fathership:father
+    Then transaction commits
+    When connection close all sessions
+    When connection open data session for database: typedb
+    When session opens transaction of type: write
+    Then $m = relation(fathership) create new instance
+    When $a = entity(person) create new instance
+    When relation $m add player for role(father): $a
```

Comment:
we create a relation subtype and insert an instance

Reply:
I feel like this scenario is getting very long and it's hard to figure out what it's actually testing because it's a very long block of text.

And it feels like some of the lines are superfluous already, for example `Then relation(parentship) create new instance; throws exception` on line 215, is of course going to throw because we opened a `READ` transaction, that is surely out of scope of this scenario.

Maybe we can clean it up a little and perhaps break it down?

Reply:
done!